resources/sdk/pureclouddotnet/templates/ApiClient.mustache

```diff
@@ -113,7 +113,7 @@ namespace {{packageName}}.Client
         /// Gets or sets the RestClient.
         /// </summary>
         /// <value>An instance of the RestClient</value>
-        public RestClient RestClient { get; set; }
+        private RestClient RestClient { get; set; }
```

Comment:
I am a little concerned about restricting access to this.  Usually, we would let the rest client in an API be exposed so that if there is a corner case that the API can not handle, the customer might still be able to do the work with RestClient.  Is there a specific reason why we decided to change this from public to private?  

Thanks,
     John